contracts/staking/PoSValidatorManager.sol

```diff
@@ -39,6 +58,11 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
     bytes32 private constant _POS_VALIDATOR_MANAGER_STORAGE_LOCATION =
         0x4317713f7ecbdddd4bc99e95d903adedaa883b2e7c2551610bd13e2c7e473d00;
 
+    uint8 public constant MAXIMUM_STAKE_MULTIPLIER_LIMIT = 10;
+
+    // TODO: revisit maximum, currently set to 100%
```

Comment:
I think 100% makes sense. A validator would set this if they only want to delegate to themselves